scripts/build-image.sh

```diff
@@ -122,6 +124,16 @@ if [[ "$CLOUD_PROVIDER" == "azure" ]] && [[ "$NO_IMAGE" = false ]]; then
     echo "OK"
 fi
 
+if [[ "$CLOUD_PROVIDER" == "gce" ]] && [[ "$NO_IMAGE" = false ]]; then
+    REQUIRED_PROGRAMS="$REQUIRED_PROGRAMS gsutil gcloud jq"
```

Comment:
Do we overwrite this below?  Probably need to update the `REQUIRED_PROGRAMS="qemu-img qemu-nbd"` to be like you have it here, `REQUIRED_PROGRAMS="$REQUIRED_PROGRAMS qemu-img qemu-nbd"`

Reply:
Oh that is a very good point, let me make that update

Reply:
fixed in my recent commit.